### DISCUSSION TITLE
How to make a profile only for music? Condition=no video stream

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#14671] [Q&A] [answered]
Hi, I want to make a profile only for music. So a condition when there is no video stream would be needed.

Thank you :)

**Comment by guidocella** [comment, marked as answer]
https://github.com/mpv-player/mpv/wiki/FAQ#user-content-How_do_I_create_a_Conditional_auto_profile_that_applies_to_all_videos_or_all_audio

**Reply by geextahslex** [reply]
Thank you :)
